app/config/tstvhq/rooms.json

```diff
@@ -494,8 +495,7 @@
             "doorTint": "white",
             "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/crematorium.jpg",
             "directions": {
-                "west": "secretTunnels"
-            }
+                "west": "secretTunnels" }e
```

Comment:
should be on a new line below (same indentation as "directions"), no trailing `e`